jdi-light/src/main/java/com/epam/jdi/light/settings/WebSettings.java

```diff
@@ -158,41 +110,12 @@ public static synchronized void init() {
         try {
             getProperties(COMMON.testPropertiesPath);
-            fillAction(p -> COMMON.strategy = getStrategy(p), "strategy");
-            COMMON.strategy.action.execute();
-            fillAction(p -> TIMEOUTS.element = new Timeout(parseInt(p)), "timeout.wait.element");
-            fillAction(p -> TIMEOUTS.page = new Timeout(parseInt(p)), "timeout.wait.page");
-            fillAction(WebSettings::setDomain, "domain");
-            if (DRIVER.name.equals(DEFAULT_DRIVER))
-                fillAction(p -> DRIVER.name = p, "driver");
-            fillAction(p -> DRIVER.version = p, "driver.version");
-            fillAction(p -> DRIVER.path = p, "drivers.folder");
-            fillAction(p -> SCREEN.path = p, "screens.folder");
-            fillAction(p -> ELEMENT.startIndex = parseInt(p), "list.start.index");
-            addStrategy(FAIL, LOGS.screenStrategy);
-            fillAction(p -> LOGS.logInfoDetails = getInfoDetailsLevel(p), "log.info.details");
-            fillAction(p -> LOGS.screenStrategy = getLoggerStrategy(p), "screenshot.strategy");
-            fillAction(p -> LOGS.htmlCodeStrategy = getLoggerStrategy(p), "html.code.strategy");
-            fillAction(p -> LOGS.requestsStrategy = getLoggerStrategy(p), "requests.strategy");
-            fillAction(p -> COMMON.killBrowser = p, "browser.kill");
-            fillAction(WebSettings::setSearchStrategy, "element.search.strategy");
-            fillAction(p -> DRIVER.screenSize.read(p), "browser.size");
-            fillAction(p -> DRIVER.pageLoadStrategy = getPageLoadStrategy(p), "page.load.strategy");
-            fillAction(p -> PAGE.checkPageOpen = parse(p), "page.check.after.open");
-            fillAction(SoftAssert::setAssertType, "assert.type");
-            fillAction(p -> ELEMENT.clickType = getClickType(p), "click.type");
-            fillAction(p -> ELEMENT.getTextType = getTextType(p), "text.type");
-            fillAction(p -> ELEMENT.setTextType = getSetTextType(p), "set.text.type");
-            // RemoteWebDriver properties
-            fillAction(p -> DRIVER.remoteUrl = getRemoteUrl(p), "remote.type");
-            fillAction(p -> DRIVER.remoteUrl = p, "driver.remote.url");
-            fillAction(p -> LOGS.logLevel = parseLogLevel(p), "log.level");
-            logger.setLogLevel(LOGS.logLevel);
-            fillAction(p -> LOGS.writeToAllure = parseBoolean(p), "allure.steps");
-            fillAction(p -> ELEMENT.smartTemplate = p.split(";")[0], "smart.locators");
-            fillAction(p -> ELEMENT.smartName = getSmartSearchFunc(p), "smart.locators.toName");
-            fillAction(p -> ELEMENT.useSmartSearch = getSmartSearchUse(p), "smart.search");
-            fillAction(p -> DRIVER.capabilities.common.put("headless", p), "headless");
+            TestProperties.getTestsProperties().forEach((k,v)->
+                                {String name = k;
+                                JAction1<String> action1 = v.key;
```

Comment:
give action and action1 more useful names

Reply:
done